packages/cmf/src/middlewares/http/middleware.js

```diff
@@ -86,6 +86,7 @@ export function HTTPError(response) {
 	this.name = `HTTP ${response.status}`;
 	this.message = response.statusText;
 	this.stack = {
+		response,
```

Comment:
I need this to handle this in the onHTTPError. If the text() sucess, it crushes it with the serialization.